app/models/document_capture_session.rb

```diff
@@ -18,8 +18,8 @@ def store_result_from_response(doc_auth_response)
     session_result.attention_with_barcode = doc_auth_response.attention_with_barcode?
     session_result.selfie_check_performed = doc_auth_response.selfie_check_performed?
     session_result.doc_auth_success = doc_auth_response.doc_auth_success?
-    # nil(selfie not required) or true/false
-    session_result.selfie_success = doc_auth_response.selfie_success
+    session_result.selfie_status = doc_auth_response.respond_to?(:selfie_status) ?
```

Comment:
@eileen-nava - whenever reading the selfie_status from DocumentCaptureResult, a string will be retrieved, yes? is there a way to return a symbol when retrieving the selfie_status attribute from DocumentCaptureResult?

Reply:
Though it's debatable to include complex behaviors with struct, which basically a data holder.

Like hash in ruby, many cases it get overuses it if not abuses.

Reply:
a bit simpler ... i tried defining the method #selfie_status  as follows in DocumentCaptureSessionResult which seemed to work ...
```
    def selfie_status
      self[:selfie_status].to_sym
    end
```
obs document_capture_session_result[:selfie_status] will still return a string, but using the member to fetch is not stylistically the way we're accessing attributes in this struct 🤔

Reply:
@amirbey , that works too.

Reply:
@dawei-nava @amirbey I implemented the first suggestion, then refreshed this thread and saw the second one.  Is there a strong preference between the two?  I agree that both work.

Reply:
@eileen-nava , no!

Reply:
@eileen-nava , we need to consider accessing using result[:selfie_status] or result['selfie_status'] method calls
For example(draft):
```ruby
def [](key)
    return super.to_sym if key.to_s == :test_prop
    super
end
```

Reply:
> @dawei-nava @amirbey I implemented the first suggestion, then refreshed this thread and saw the second one. Is there a strong preference between the two? I agree that both work.

I agree both work ... i prefer the latter approach for reasons stated here ... https://github.com/18F/identity-idp/pull/9953/files#r1465155665